crown-court-proceeding/src/test/java/uk/gov/justice/laa/crime/crowncourt/controller/CrownCourtProceedingIntegrationTest.java

```diff
@@ -202,6 +207,78 @@ void givenAValidContent_whenUpdateApplicationIsInvoked_thenUpdateApplicationIsSu
         assertThat(result.getResponse().getContentAsString()).isEqualTo(objectMapper.writeValueAsString(updateApplicationResponse));
     }
 
+
+    @Test
+    void givenAEmptyOAuthToken_whenUpdateIsInvoked_thenFailsUnauthorizedAccess() throws Exception {
+        mvc.perform(RequestBuilderUtils.buildRequestGivenContent(
+                        HttpMethod.PUT, "{}", ENDPOINT_URL + UPDATE_CC_URL, Boolean.FALSE))
+                .andExpect(status().isUnauthorized());
+    }
+
+    @Test
+    void givenAInvalidContent_whenUpdateIsInvoked_thenFailsBadRequest() throws Exception {
+        mvc.perform(RequestBuilderUtils.buildRequestGivenContent(
+                        HttpMethod.PUT, objectMapper.writeValueAsString(
+                                TestModelDataBuilder.getApiUpdateApplicationRequest(Boolean.FALSE)), ENDPOINT_URL + UPDATE_CC_URL))
+                .andExpect(status().isBadRequest());
+    }
```

Comment:
Do we really need these tests? They seem to just be duplicates of what we have in the controller test class.

Reply:
Thanks. Reused this test case for ValidationException